flink-core/src/main/java/org/apache/flink/configuration/WebOptions.java

```diff
@@ -89,78 +95,89 @@
 	public static final ConfigOption<String> UPLOAD_DIR =
 		key("web.upload.dir")
 			.noDefaultValue()
-			.withDeprecatedKeys("jobmanager.web.upload.dir");
+			.withDeprecatedKeys("jobmanager.web.upload.dir")
+			.withDescription("The config parameter defining the directory for uploading the job jars. If not" +
+				" specified a dynamic directory will be used under the directory specified by JOB_MANAGER_WEB_TMPDIR_KEY.");
 
 	/**
 	 * The config parameter defining the number of archived jobs for the jobmanager.
 	 */
 	public static final ConfigOption<Integer> ARCHIVE_COUNT =
 		key("web.history")
 			.defaultValue(5)
-			.withDeprecatedKeys("jobmanager.web.history");
+			.withDeprecatedKeys("jobmanager.web.history")
+			.withDescription("The config parameter defining the number of archived jobs for the jobmanager.");
 
 	/**
 	 * The log file location (may be in /log for standalone but under log directory when using YARN).
 	 */
 	public static final ConfigOption<String> LOG_PATH =
 		key("web.log.path")
 			.noDefaultValue()
-			.withDeprecatedKeys("jobmanager.web.log.path");
+			.withDeprecatedKeys("jobmanager.web.log.path")
+			.withDescription("Define the log file location (may be in /log for standalone but under log directory when using YARN).");
```

Comment:
let's get some consistency in here. Some descriptions start with `The config parameter`, others with `Config parameter defining`, and others with `Define`.

I know you just copied the javadocs, but imo this isn't sufficient for the docs.

Reply:
Thanks for the advise.

Reply:
Some examples:
`parallelism.default`: "Default parallelism for jobs."
`metrics.scope.delimiter`: "Delimiter used to assemble the metric identifier."

Basically, remove repetitive prefixes that could be used for every description.